frontend/src/components/sign/components/form-sign-up.tsx

```diff
@@ -0,0 +1,140 @@
+import * as React from 'react';
+import { useForm } from "react-hook-form";
+import { yupResolver } from '@hookform/resolvers/yup';
+
+import styles from './styles.module.scss';
+
+import { RegisterPayloadKey, IRegisterPayload, UserType, UserSex } from 'healthcare-shared'
+import { validationUserSchema } from 'validation-schema'
```

Comment:
```suggestion
import { validationUserSchema } from 'validation-schemas'
```
It isn't user validation schema. It's registration schema